pkg/workspace/manifests/manifests.go

```diff
@@ -254,7 +253,6 @@ func GenerateTuningJobManifest(ctx context.Context, wObj *kaitov1beta1.Workspace
 			},
 		},
 		Spec: batchv1.JobSpec{
-			BackoffLimit: &numBackoff, // default is 6. A failed tuning job is unlikely to be self-recoverable, no need to recreate the pod.
```

Comment:
If we could save past jobs (including their data), we could enable backoff retries with or without GRIT.

Reply:
This change makes sense in general. We should consider other approaches for saving intermittent data.